### ISSUE TITLE
Version 0.2.0 benchmarks can not reproduced with current master using standard config file

### ISSUE THREAD

**Issue by TjarkMiener** [ctlearn-project/ctlearn#84]
I had problems reproducing the benchmarks (version 0.2.0; single tel LST) with the current master using a standard config file (see above). Bryan runs into the same problem on shevek. The run finish after ~20 mins/15000 step with accuracy (LST single_tel): INFO:tensorflow:Saving dict for global step 14955: accuracy = 0.59607786, accuracy_gamma = 0.3556767, accuracy_proton = 0.78488666, auc = 0.59984684, global_step = 14955, loss = 1.3337153 

config parameters:
Data:
  Input:
    batch_size: 64
    map: true
    num_parallel_calls: 2
    prefetch: true
    prefetch_buffer_size: 1
    shuffle: true
    shuffle_buffer_size: 10000
  Loading:
    example_type: single_tel
    min_num_tels: 1
    seed: 1234
    selected_tel_types: ['LST:LSTCam']
    selected_tel_ids:
      'LST:LSTCam': [1, 2, 3, 4]
    validation_split: 0.1
  Processing:
    sorting: null
  file_list: /home/tjark/prototype_files_class_balanced.txt
  format: HDF5
Image Mapping:
  hex_conversion_algorithm: 'oversampling'
  padding:
    LSTCam: 2
    FlashCam: 1
    NectarCam: 2
    SCTCam: 4
    DigiCam: 1
    ASTRICam: 0
    CHEC: 0
    VERITAS: 1
Logging:
  model_directory: /home/tjark/logs/hex_conversion_algorithm_test/oversampling/LST_single_tel_config
Model:
  Model Parameters:
    basic:
      conv_block:
        batchnorm: false
        bottleneck: null
        layers:
        - filters: 32
          kernel_size: 3
        - filters: 32
          kernel_size: 3
        - filters: 64
          kernel_size: 3
        - filters: 128
          kernel_size: 3
        max_pool:
          size: 2
          strides: 2
    cnn_rnn:
      cnn_block:
        function: conv_block
        module: basic
      dropout_rate: 0.5
      pretrained_weights: null
    single_tel:
      network:
        function: conv_block
        module: basic
      pretrained_weights: null
  model:
    function: single_tel_model
    module: single_tel
  model_directory: /home/tjark/ctlearn/ctlearn/default_models
Prediction:
  export_as_file: true
  prediction_file_path: /home/tjark/predictions/predictions.csv/prediction_LST_single_tel_config.csv
  true_labels_given: true
TensorFlow:
  run_TFDBG: false
Training:
  Hyperparameters:
    adam_epsilon: 1.0e-08
    apply_class_weights: true
    base_learning_rate: 5.0e-05
    optimizer: Adam
    scale_learning_rate: false
    variables_to_train: null
  num_training_steps_per_validation: 2500
  num_validations: 15

**Comment by aribrill** [comment]
Looks like the run is terminating before the full number of training steps is completed. It should end after 2500 x 15 = 37500 steps. I confirmed this was the case for the v0.2.0 benchmark runs. Here there are only 14955 steps. Tjark, Bryan, do you both get this same number? Does each epoch have 14955 / 15 = 997 steps, or does the run end during epoch 6 at 11964 steps?

**Comment by TjarkMiener** [comment]
Bryan texted me "At step ~15000 I got identical results to you". Did the run finished after ~15000 steps? I got 14955 / 15 = 997 steps per epoch.

**Comment by bryankim96** [comment]
Yes, I got the same. I used the same random seed, the results were completely identical

**Comment by TjarkMiener** [comment]
This commit https://github.com/ctlearn-project/ctlearn/commit/4f6946b406ae9678c605ebf9577a1f82afde7182 breaks it.
